packages/firebase_ml_vision/android/src/main/java/io/flutter/plugins/firebasemlvision/CloudLabelDetector.java

```diff
@@ -52,26 +56,4 @@ public void onFailure(@NonNull Exception e) {
               }
             });
   }
-
-  private FirebaseVisionCloudDetectorOptions parseOptions(Map<String, Object> optionsData) {
```

Comment:
I moved this method to `FirebaseMLVisionPlugin.java` so it can be used by all cloud detectors.